src/lib/source-handlers/azure/index.ts

```diff
@@ -0,0 +1,3 @@
+export { listAzureProjects } from './list-projects';
+export { AzureRepoData } from './types';
+export { listAzureRepos, fetchAllRepos } from './list-repos';
```

Comment:
why is `fetchAllRepos` exported?

Reply:
My bad, forgot to remove it. Will remove

Reply:
Done